### PR TITLE
Social media adapter

### PR DESCRIPTION
**Pull request by serhii-kutsyi**

### CODE REVIEW

**Comment by serhii-kutsyi** [comment]
@ion-agena please review

File: api-gateway/api-gateway-kubernetes.yml [hunk 1]
```diff
@@ -12,10 +12,17 @@ spring:
         - id: profilesDataPrincipal
           uri: http://profiles
           predicates:
-            - Path=/api/v1/influencers/me/followersCount, /api/v1/influencers/me/externalReferences
```
Comment (ion-agena) [inline api-gateway/api-gateway-kubernetes.yml:15]: why the followers count was removed?

**Review by ion-agena — COMMENTED** [review]
followers count endpoint should not be removed

File: api-gateway/api-gateway-kubernetes.yml (hunk 1 above)
Comment (serhii-kutsyi) [reply api-gateway/api-gateway-kubernetes.yml:15]: this operation was moved to the social media adapter.

Comment (ion-agena) [reply api-gateway/api-gateway-kubernetes.yml:15]: social media adapter should perform a call to the campaign manager to save the actual number of followers in our db (req no 8):
https://agena-tech.atlassian.net/wiki/spaces/INF/pages/19890200/Set+Followers+Count

the endpoint above should be used for that

**Review by ion-agena — APPROVED** [review]

**Pull request merged**